tests/test_indexer.py

```diff
@@ -461,6 +490,7 @@ def do_POST(self):
     def reset(cls):
         cls._response_code = 200
         cls._payload = None
+        cls.authenticated_requests = 0
```

Comment:
where is this used?

Reply:
Sorry, that was unused and not cleaned up. Fixed.